PackageViewModel/CredentialManagerProvider.cs

```diff
@@ -23,16 +26,16 @@ public Task<CredentialResponse> GetAsync(Uri uri, IWebProxy proxy, CredentialReq
         {
             if (isRetry)
             {
-                return Task.FromResult(new CredentialResponse(CredentialStatus.ProviderNotApplicable));
+                return Task.FromResult(new CredentialResponse(CredentialStatus.UserCanceled));
             }
 
-            var credentials = _credentialManager.Value.Get(uri);
+            var credentials = _credentialManager.Get(uri);
 
             if (credentials != null)
             {
                 return Task.FromResult(new CredentialResponse(credentials));
             }
-            return Task.FromResult(new CredentialResponse(CredentialStatus.ProviderNotApplicable));
+            return Task.FromResult(new CredentialResponse(CredentialStatus.UserCanceled));
```

Comment:
Using `UserCanceled` here to allow the CredentialProvider to be called multiple times. Even if this time there are no credentials. When using `ProviderNotApplicable` then the CredentialProvider is never used again for this feed.